cam_width: 640
cam_height: 480
cam_fx: 347.997558
cam_fy: 347.997558
cam_cx: 320.0
cam_cy: 240.0

